dnd5e.mjs

```diff
@@ -84,6 +84,8 @@ Hooks.once("init", function() {
   CONFIG.DND5E = DND5E;
   CONFIG.ActiveEffect.documentClass = documents.ActiveEffect5e;
   CONFIG.Actor.documentClass = documents.Actor5e;
+  CONFIG.Actor.systemDataModels.group = documents.GroupActor;
```

Comment:
The item and advancement data models we've implemented so far in `2.1.x` have taken a slightly different organization approach, separating the system data models from the documents in their own folder.